user/caribou/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ get_filename_component(EUDAQ_USERNAME ${CMAKE_CURRENT_LIST_DIR} NAME)
 set(EUDAQ_MODULE module_${EUDAQ_USERNAME})
 set(USER_HARDWARE_LIBRARY ${EUDAQ_USERNAME}_static)
 
+# DSO9254A converter needs root
```

Comment:
An alternative to *always* requiring ROOT would be to only build the DSO converter if ROOT is found.
But I actually think it's fine this way, otherwise things get overly complicated.